docs/cli-reference/dfx-canister.md

```diff
@@ -108,6 +108,7 @@ You can use the following options with the `dfx canister call` command.
 
 | Option                  | Description  |
 |-------------------------|--------------|
+| `--argument-file`       | Specifies the file from which to read the argument to pass to the method.  |
```

Comment:
FYI I also added this. This will then get pulled over to https://internetcomputer.org/docs/current/references/cli-reference/dfx-canister

Reply:
Thank you.